### ISSUE TITLE
Fix could not set parameter error for test : tests.foreman.ui.test_operatingsys.OperatingSys.test_[remove_os, remove_os_parameter, negative_set_os_parameter_2]

### ISSUE THREAD

**Issue by jyejare** [SatelliteQE/robottelo#2040]
Feel that adding a double inverted commas to OS parameter while searching through search box makes this test fail. Removing the double inverted commas should do the trick.

StackTrace:

```
Traceback (most recent call last):
  File "/home/jyejare/Desktop/RedHat/RobotteloTop/robottelo/tests/foreman/ui/test_operatingsys.py", line 456, in test_remove_os_parameter
    self.fail(e)
AssertionError: Could not set parameter 'NWOG'
-------------------- >> begin captured logging << --------------------
py.warnings: WARNING: /usr/lib/python2.7/site-packages/requests/packages/urllib3/util/ssl_.py:79: InsecurePlatformWarning: A true SSLContext object is not available. This prevents urllib3 from configuring SSL appropriately and may cause certain SSL connections to fail. For more information, see https://urllib3.readthedocs.org/en/latest/security.html#insecureplatformwarning.
  InsecurePlatformWarning

py.warnings: WARNING: /usr/lib/python2.7/site-packages/requests/packages/urllib3/connectionpool.py:769: InsecureRequestWarning: Unverified HTTPS request is being made. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.org/en/latest/security.html
  InsecureRequestWarning)

root: DEBUG: NoSuchElementException: Could not locate element //div[contains(@class, 'jnotify-notification-error')].
root: DEBUG: TimeoutException: Timed out waiting for element '//a[contains(., '𡂆𠇰ﲣ𪢋캁䙆榼ꃬ矅쯵諿㔮澏𨼩剝ꙓ𤱃왐𪄘𤠘饢𠜉𡭭园𧞀𠵕쭱𣛈ꔊ𥳴')]' to display.
--------------------- >> end captured logging << ---------------------
```

**Comment by jyejare** [comment]
Another test observed with same issue: 
Test name: tests.foreman.ui.test_operatingsys.OperatingSys.test_remove_os

**Comment by jyejare** [comment]
One more test observed with same issue:
Test name: tests.foreman.ui.test_operatingsys.OperatingSys.test_negative_set_os_parameter_2

**Comment by jyejare** [comment]
Closing the issue as : 
1. The fix for test has been taken care into merged PR #2061 .
2. The test remove_os_parameter is passing without any failure on my local setup on both RHEL 6 and RHEL 7. Also the test is passing in nightly.
3. The test negative_set_os_parameter_2 is passing without any failure on my local setup on both RHEL 6 and RHEL 7. Also the test is passing in nightly.

So closing this issue.
